test/unit/data/test-structure-data.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable */
+module.exports =
+[
```

Comment:
After addressing [this comment](https://github.com/uberVU/mocha-mugshot-reporter/pull/4/files#r39857212) I'm guessing you won't need this file anymore.